lambdatimes/src/components/Content/Content.js

```diff
@@ -18,10 +18,17 @@ export default class Content extends Component {
 
   componentDidMount() {
     // Once the component has mounted, get the data and reflect that data on the state.
+    this.setState({
+      tabs: tabData,
+      cards: cardData
+    })
   }
 
   changeSelected = tab => {
     // this function should take in the tab and update the state with the new tab.
+    this.setState({
+      selected: tab.target.innerText.toLowerCase(),
+    })
```

Comment:
If I'm not mistaken, can you accomplish the same thing doing `this.setState({ selected: tab })` ?